src/app/admin/users/page.tsx

```diff
@@ -15,32 +15,37 @@ import {
   Table,
 } from '@/components/ui/table';
 import {
-  PaginationPrevious,
-  PaginationItem,
-  PaginationLink,
-  PaginationEllipsis,
-  PaginationNext,
   PaginationContent,
   Pagination,
+  PaginationPages,
 } from '@/components/ui/pagination';
-import { SVGProps, useEffect } from 'react';
+import { SVGProps, useEffect, useState } from 'react';
 import findUsers from '@/services/user/find';
+import { UserDto } from '@/services/user/dto/user.dto';
+import { CheckCircle2, XCircle } from 'lucide-react';
 
 type TSVGElementProps = SVGProps<SVGSVGElement>;
 
 const UserList = () => {
+  const [users, setUsers] = useState<UserDto[] | null>(null);
```

Comment:
You can init the state as an empty array instead of `null`, this will allow you to do users.map instead of users?.map in the table.

Reply:
Ty, that will make code nicer